sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/command/AlterTableAddPartitionSuite.scala

```diff
@@ -36,11 +36,11 @@ class AlterTableAddPartitionSuite
           sql(s"INSERT INTO $t PARTITION (part=0) SELECT 0")
           assert(!statsOn || getTableSize(t) > 0)
 
-          checkHiveClientCalls(expected = 17) {
+          checkHiveClientCalls(expected = 23) {
```

Comment:
We need such change because we now call `executePlan` in `catalog.refreshTable`. This is similar to other changes.